<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Janela</class>
 <widget class="QMainWindow" name="Janela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>450</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Golpe de Ariete - DEMO</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Desktop/Icone.png</normaloff>../../../Desktop/Icone.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="PlotArea" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>250</height>
       </size>
      </property>
      <property name="title">
       <string>Controles</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="4" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QComboBox" name="comboBox_Material">
           <item>
            <property name="text">
             <string>Aço</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ferro fundido</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ferro dúctil</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PVC</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_Material">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Material da tubulação</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="5">
        <widget class="QPushButton" name="pushButton_Parar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Parar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_Dispositivo">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Dispositivo</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QComboBox" name="comboBox_Dispositivos">
           <item>
            <property name="text">
             <string>Nenhum</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Chaminé de equilíbrio</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reservatório hidropneumático</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>TAU</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Válvula de Retenção</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ventosa</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_Posicao">
           <property name="text">
            <string>Posição:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBox_Posicao">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>19</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" rowspan="3">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_Q">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Vazão (m³/s)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DoubleSlider" name="slider_Vazao">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>101</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_valor_Q">
             <property name="text">
              <string>0.3</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_WR2">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>WR² (kgf.m²)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DoubleSlider" name="slider_WR2">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>101</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_valor_WR2">
             <property name="text">
              <string>0.01</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="pushButton_Rodar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Rodar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_Tempo">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Tempo de simulação (s)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DoubleSlider" name="slider_Tempo">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>101</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_valor_Tempo">
           <property name="text">
            <string>200</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlotArea</class>
   <extends>QGraphicsView</extends>
   <header>plotarea.h</header>
  </customwidget>
  <customwidget>
   <class>DoubleSlider</class>
   <extends>QSlider</extends>
   <header>doubleslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
